tests/load/locustfiles/locustfile.py

```diff
@@ -31,119 +48,392 @@ def _(parser: Any):
         required=True,
         help="Endpoint URL",
     )
+    parser.add_argument(
+        "--wait_time",
+        type=str,
+        env_var="AUTOPUSH_WAIT_TIME",
+        help="AutopushUser wait time between tasks",
+        default="20, 25",
+    )
 
 
-class TimeEvent:
-    def __init__(self, user: FastHttpUser, name: str) -> None:
-        self.user: FastHttpUser = user
-        self.start_time: float
-        self.name: str = name
-
-    def __enter__(self) -> object:
-        self.start_time = time.perf_counter()
-        self.response_length: int = 0
-        return self
-
-    def __exit__(self, *args) -> None:
-        end_time: float = time.perf_counter()
-        exception: Any = None
-        if args[0] is not None:
-            exception = args[0], args[1]
-
-        if not isinstance(args[1], (AssertionError, type(None))):
-            self.user.environment.events.user_error.fire(
-                user_instance=self.user.context(), exception=args[1], tb=args[2]
-            )
-            exception = None
-        self.user.environment.events.request.fire(
-            request_type="WSS",
-            name=self.name,
-            response_time=(end_time - float(str(self.start_time))) * 1000,
-            response_length=self.response_length,
-            exception=exception,
-            context=self.user.context(),
-        )
+@events.test_start.add_listener
+def _(environment, **kwargs):
+    environment.autopush_wait_time = parse_wait_time(environment.parsed_options.wait_time)
 
 
 class AutopushUser(FastHttpUser):
-    wait_time = between(30, 35)
+    REST_HEADERS: dict[str, str] = {"TTL": "60", "Content-Encoding": "aes128gcm", "Topic": "aaaa"}
```

Comment:
A "Topic" header means that subsequent subscription updates will replace the undelivered update. (e.g. if you have a pending 'aaaa' topic update, the new 'aaaa' update will replace it. This means that the UA will receive only the latest 'aaaa' update, instead of the default behavior, which is to receive ALL pending updates.)

Is this intended?

(Even if it is, might be worth adding a comment describing this behavior for future folk that might not be aware of it.)

Reply:
I think this might explain why my numbers don't add up during test runs! TY!

I actually don't know the original intent behind having the topic here, but my gut reaction from your description is that for load test purposes we should remove this.